extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php

```diff
@@ -3,7 +3,7 @@
 class StaffWelcomePoster {
 
 	const LANG_TO_STAFF_MAP = [
-		'de' => 24961680,      // ForestFairy
+		'de' => 26339491,      // Mira Laime
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)